internal/pkg/service/common/task/node_test.go

```diff
@@ -643,12 +644,16 @@ func TestTaskTimeout(t *testing.T) {
 		ProjectID: 123,
 		TaskID:    task.ID("my-receiver/my-export/" + taskType),
 	}
-	tel := telemetry.NewForTest(t)
 
-	// Create node and start task
+	tel := newTestTelemetryWithFilter(t)
+
+	// Create node and
 	node1, d := createNode(t, etcdCredentials, nil, tel, "node1")
 	logger := d.DebugLogger()
+	logger.Truncate()
 	tel.Reset()
```

Comment:
Testy som zjednotil, niekde chybal `logger.Truncate()` alebo `tel.Reset()` po vytvoreni nodes.
Teraz je to rovnako, .... operacie pocas setup-u nas v teste nezaujimaju.